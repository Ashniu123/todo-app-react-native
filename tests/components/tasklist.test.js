import React from 'react';
import { shallow } from 'enzyme';

import { Tasklist } from '../../src/components/taskslist/tasklist.component';

test('should render Tasklist component correctly', () => {
	const wrapper = shallow(<Tasklist />);

	expect(wrapper).toMatchSnapshot();
});