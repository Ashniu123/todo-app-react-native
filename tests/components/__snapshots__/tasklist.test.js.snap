// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Tasklist component correctly 1`] = `
<Component
  style={Object {}}
>
  <FlatList
    disableVirtualization={false}
    horizontal={false}
    initialNumToRender={10}
    keyExtractor={[Function]}
    maxToRenderPerBatch={10}
    numColumns={1}
    onEndReachedThreshold={2}
    renderItem={[Function]}
    scrollEnabled={true}
    scrollEventThrottle={50}
    updateCellsBatchingPeriod={50}
    windowSize={21}
  />
</Component>
`;
